tests/unittests/net/artifacts/photon_net_config_v2.yaml

```diff
@@ -0,0 +1,82 @@
+#cloud-config
+network:
+  version: 2
+
+  ethernets:
+    eth0:
+      match:
+        macaddress: "00:11:22:33:44:55"
+      set-name: eth0
+      dhcp4: false
+
+    eth1:
+      dhcp4: true
+      dhcp6: true
+
+    eth2:
+      match:
+        macaddress: "00:aa:bb:cc:dd:ee"
+      set-name: eth2
+      dhcp4: false
+      addresses:
+        - 10.20.30.40/24
+      nameservers:
+        addresses:
+          - 9.9.9.9
+          - 8.8.8.8
+        search:
+          - example.org
+
+  bonds:
+    bond0:
+      interfaces: [eth0, eth2]
+      parameters:
+        mode: 802.3ad
+        mii-monitor-interval: 100
+        transmit-hash-policy: layer2+3
+        lacp-rate: fast
+      dhcp4: false
+      dhcp6: false
+
+  vlans:
+    vlan100:
+      id: 100
+      link: eth0
+      addresses:
+        - 192.168.100.10/24
+        - 192.168.100.11/24
+      dhcp6: true
+      mtu: 1500
```

Comment:
`mtu: 1500` is repeated in two different places in this file. Can we use a different value for each to ensure the right value is going to the right file?